tests/e2e/utils/features.js

```diff
@@ -33,7 +33,10 @@ export async function disableFeature( feature ) {
 	return await wpApiFetch( {
 		path: 'google-site-kit/v1/e2e/feature/set-flag',
 		method: 'post',
-		data: { [ feature ]: false },
+		data: {
+			feature_name: feature,
+			feature_value: false,
+		},
```

Comment:
Nice catch. We should have probably required this to be used in the PR somehow to test 👍